tracer/src/Datadog.Trace/AppSec/Transports/Sender.cs

```diff
@@ -41,12 +41,7 @@ internal async Task Send(IEnumerable<IEvent> events)
             };
             var request = _apiRequestFactory.Create(_uri);
             request.AddHeader(AppSecHeaderKey, AppSecHeaderValue);
-            var response = await request.PostAsJsonAsync(batch, _serializer);
-            if (response.StatusCode != 200 && response.StatusCode != 202)
-            {
-                var responseText = await response.ReadAsStringAsync();
-                Log.Warning($"AppSec event not correctly sent to backend {response.StatusCode} with response {responseText}");
-            }
+            return request.PostAsJsonAsync(batch, _serializer);
```

Comment:
Not passionate about it, but you _might_ still want to await this one to get a better stack trace if it errors?

Reply:
I thought if you await at caller level, the exception bubbles up fine.. Although PostAsJsonAsync returns a Task<IApiResponse> and the caller doesn't use the result yet, just awaiting it. So I'm not sure in this case... 😅 
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1778/commits/62c946f8890c5547eb463e90479a50f783e369bf

Reply:
Yeah the exception bubbles up fine either way, but you get a slightly screwy stack trace. For example if `PostAsJsonAsync()` throws an exception, and it isn't awaited here, `Send()` won't appear on the stack trace. But it's less efficient to add the await, so it's a trade off 😉

Reply:
oh ok thanks for the explanation :) didn't know the intermediate asynchronous methods that don't await wouldn't appear!